client/components/seo-preview-pane/index.jsx

```diff
@@ -30,6 +31,26 @@ const GooglePreview = site =>
 		snippet={ site.description }
 	/>;
 
+const PreviewFacebook = site => (
+	<div>
+		<FacebookPreview
+			title={ site.name }
+			url={ site.URL }
+			type="website"
+			description={ site.description }
+			image={ `${ get( site, 'icon.img', '//gravatar.com/avatar/' ) }?s=512` }
+		/>
```

Comment:
@roundhill why? this preview is also available from *My Sites* which shows the whole site-level preview